tests/e2e/test_rosacli_cluster.go

```diff
@@ -425,14 +425,20 @@ var _ = Describe("Edit cluster",
 		It("can verify delete protection on a rosa cluster - [id:73161]",
 			labels.High, labels.Runtime.Day2, labels.Exclude,
 			func() {
+				By("Get default delete protection value")
+				output, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				deleteProtectionDefault := output.EnableDeleteProtection
+				defaultState := fmt.Sprintf("--enable-delete-protection=%s", deleteProtectionDefault)
```

Comment:
I don't think you need a variable for that. Just put it into the defer method call